packages/localization/package.json

```diff
@@ -8,11 +8,11 @@
     "@kiwicom/rnmodules": "^0",
     "date-fns": "^1.30.1",
     "form-data": "^2.5.1",
+    "@adeira/fetch": "^1.0.0",
```

Comment:
Do you want it as a dep and not as a peerDep like previously?

Reply:
Yes, it is a dependency, and should be just that instead of peerDependency